lib/base/novacom.js

```diff
@@ -919,7 +924,8 @@ const fs = require('fs'),
                     write[extended].bind(obj[extended])(data);
                 }).stderr.on('data', function(data) {
                     log.verbose('Session#run()', "on data (stderr)");
-                    write.stderr.bind(obj.stderr)(data);
+                    orgErrMsg = data.toString();
+                    // write.stderr.bind(obj.stderr)(data);
```

Comment:
Please check there is no side effect.

Reply:
When I checked, there was no side effect.